docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+
+### Billing requests
+
+Invoices must be created using the `XRechnung` standard. There might also be dedicated accounting/book-keeping software which allows creating and sending invoices according to this standard, so that billing might also be handled outside the production system. However, the client supports sending invoices by mail. It is up to the users of the client to define what should happen with the response from the diga api. E.g. invoices are likely to be required for accounting and therefore must be stored. Also handling of apis which do not support billing yet, must be defined.
+
+The invoice should contain the following information:
+
+- prescription code
+- DiGA id of the manufacturer
+- time period for prescription
+- information about the invoice issuer and the insurance company
+
+Additional information can be found on this [wiki page](https://github.com/alex-therapeutics/diga-api-client/wiki/Billing-validation---XRechnung).
+
+### Encryption
+
+Given that we are dealing with patient data, it is required to properly encrypt the data sent within requests.
+The encryption is based on certificates which need to be requested at the itsg which acts as a trust center.
+You can request it [here](https://www.itsg.de/produkte/trust-center/zertifikat-beantragen/), assuming that you already have an IK number.
+
+A technical documentation for the encryption (of course in German) can be found [here](https://www.gkv-datenaustausch.de/media/dokumente/standards_und_normen/technische_spezifikationen/Anlage_16.pdf).
+Once the certificates are issued, checkout the [prerequisites](https://github.com/alex-therapeutics/diga-api-client#prerequisites) within the Readme to be able to use them with the api client.
+
+### Request processing time
+
+According to the documentation, the processing time constraints are as follows:
+
+Mean value 10000[msec], 95% quantile [msec] 15000
+
+Is that really 10 seconds for a single request?
+
+## FAQ
+### What can I do if I want to make requests but my DiGA is not approved yet and I therefore do not have a diga id?
+
+Just sent some 5 digit number as part of the `nutzdaten`, e.g. `12345`.
+
+### What can I do if I dont know the IK number yet and still want to test the client?
+
+You will need an IK number to request the certificate for encryption and decryption. Therefore, it is a must-have.
+
+### Are there solutions for non-java users?
+
+So far, there are no other clients available. A solution could be to create a docker image which exposes the java client via an api. Additionally, we hope that this documentation provides enough information so that clients in other languages can be developed.
+
+### My DiGA is only valid for 30 days. Can users request a fresh code from the insurance while their existing code is still valid?
```

Comment:
>  My DiGA is only valid for 30 days.

What is meant by `DiGA` here? DiGA product, DiGA application, ...?

Reply:
DiGA means `Digitale Gesundheitsanwendung` or `digital health app`. Actually fits both :) Therefore it really means the app/service. Rewriting as `My DiGA (app) [...]`